packages/phenomic-plugin-api-related-content/src/index.js

```diff
@@ -0,0 +1,25 @@
+const flatten = require("lodash.flatten")
+
+module.exports = function() {
+  return {
+    type: "api",
+    define(api) {
+      api.get("/related/:collection/limit-:limit/*.json", async function(req, res) {
+        try {
+          const limit = parseInt(res.params.limit)
+          const post = await req.db.get([ req.params.collection ], req.params[0])
+          const lists = await Promise.all([
+            ...post.value.tags.map(tag => req.db.getList(req.params.collection, { limit: limit + 1, lt, reverse }, "tags", tag)),
+            db.getList(req.params.collection,  { limit: limit + 1 }),
+          ])
```

Comment:
yeah, should be `db`.

- `lt` -> less than
- `reverse` -> reverses the query order

they both come from https://github.com/Level/levelup